<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Downloads/c++ logo.png</normaloff>../../../../Downloads/c++ logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
background:white;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:red;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1271</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:red;
}</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Ajouter un Employee </string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(12, 12, 12);</string>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(22, 22, 22);</string>
      </property>
      <property name="text">
       <string>Prenom :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(4, 4, 4);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>170</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(8, 8, 8);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_cin">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(8, 8, 8);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(22, 22, 22);</string>
      </property>
      <property name="text">
       <string>CIN :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>240</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(22, 22, 22);</string>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>290</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(22, 22, 22);</string>
      </property>
      <property name="text">
       <string>Role :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>881</width>
        <height>461</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>0</y>
         <width>271</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../Downloads/c++ logo.png</normaloff>../../../../Downloads/c++ logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_role">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>230</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>190</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(8, 8, 8);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>270</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(22, 22, 22);</string>
       </property>
       <property name="text">
        <string>Le salaire :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>350</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;
</string>
       </property>
       <property name="text">
        <string>Generateur QR code</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_salaire">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>270</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>350</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;
</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>le_nom</zorder>
     <zorder>le_prenom</zorder>
     <zorder>le_cin</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>pushButton_7</zorder>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Gestion Des employees</string>
     </attribute>
     <widget class="QLineEdit" name="edit_prenom_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(8, 8, 8);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>-10</y>
        <width>151</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../Downloads/c++ logo.png</normaloff>../../../../Downloads/c++ logo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QTableView" name="tabemp">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>161</y>
        <width>751</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter_4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>90</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#001F26;
</string>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>90</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#001F26;
</string>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <zorder>tabemp</zorder>
     <zorder>edit_prenom_2</zorder>
     <zorder>pushButton_9</zorder>
     <zorder>pb_ajouter_4</zorder>
     <zorder>pushButton_4</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Supprimer/Modifier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>451</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="supp_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(4, 4, 4);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(22, 22, 22);</string>
       </property>
       <property name="text">
        <string>Donner l'id d'employee a supprimer :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>141</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../Downloads/c++ logo.png</normaloff>../../../../Downloads/c++ logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>431</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(22, 22, 22);</string>
       </property>
       <property name="text">
        <string>ID employee a Modifier :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(22, 22, 22);</string>
       </property>
       <property name="text">
        <string>Prenom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(12, 12, 12);</string>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(22, 22, 22);</string>
       </property>
       <property name="text">
        <string>CIN :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>240</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(22, 22, 22);</string>
       </property>
       <property name="text">
        <string>Role :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(22, 22, 22);</string>
       </property>
       <property name="text">
        <string>Le salaire :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_mod">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(4, 4, 4);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_mod">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(4, 4, 4);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom_mod">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(4, 4, 4);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cin_mod">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(4, 4, 4);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="role_mod">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(4, 4, 4);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="salaire_mod">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>280</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(4, 4, 4);</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>420</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(243, 214, 23);</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>420</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Supprimer </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Statics </string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
